src/website/app/pages/Styling/styling.md

```diff
@@ -8,6 +8,8 @@ Mineral UI is built on a design system with styles ready to go out of the box.
 
 Theming is a core concept in Mineral UI.  It is powerful feature capable of providing style customizations from the application level to the component level and should generally be your first choice when looking to make style customizations. See the [theming page](/theming) for more details.
 
+<Button primary element='a' href='/palette-demo/'>Mineral UI Palette Demo</Button>
```

Comment:
It's markdown, so our linter/prettier won't catch it, but we use double quotes to wrap string props in JSX. (Same applies to Theming.md.)

Reply:
👍

Reply:
I agree. The reference on the Theming page seemed shoehorned to me.